cli/dcoscli/data/help/node.txt

```diff
@@ -94,7 +94,7 @@ Options:
     --status
         Print diagnostics job status.
     --user=<user>
-        The SSH user [default: core].
+        The SSH user (default: core).
```

Comment:
Had to do this because docopt would always be passing the default value in case the parameter is omitted, now default is depending on config value.